### DISCUSSION TITLE
Multi layer elephant foot compensation

### DISCUSSION THREAD

**Discussion by Brace91** [OrcaSlicer/OrcaSlicer#557] [Ideas]
I'd like to see the elephant compensation act more like a chamfer.  
Ex: 
XY Elephant foot - .16
Z   Elephant foot - .32

**Comment by RandomGgames** [comment]
Elephant's foot compensation on multiple layers is an absolute life saver when tuning a printer profile. All slicers should have it to be honest. The only one I know of currently that supports it is SuperSlicer which does it like so:
![image](https://github.com/SoftFever/OrcaSlicer/assets/17207260/e66ee3f9-fc5f-4277-8dff-ba9b7792a4f9)

This is a must have. I recently gained a liking to OrcaSlicer but this is one feature I'm really hurting for and turning me off, as well as auto-slicing but that's another topic.

**Reply by Kirbo** [reply]
Doesn't it slice when you switch between "Prepare" and "Preview"?
And by pressing Tabulator on the keyboard, you can switch between these two, at least that's how it works for me..
But the elephant foot thingy would be more than welcome, I'm currently trying to deal calibrating my printer and that's the only issue left 😅

**Reply by RandomGgames** [reply]
Yes that is true, however it would be nice to see it auto-resliced every time you make a change to a profile to see the changes side by side almost. Would be helpful for comparing when an option is changed if there's less of a delay and less inputs needed.

**Reply by Kirbo** [reply]
Actually just figured out my own elephant foots with these (not sure which setting had the most impact, but now the first layer is precisely as wide as the layers above)
![image](https://github.com/SoftFever/OrcaSlicer/assets/1077974/03dcb738-70bf-46c2-a703-f564f3994d36)

Originally found from here: https://help.prusa3d.com/article/elephant-foot-compensation_114487

Perhaps also worth looking into:
- https://github.com/SoftFever/OrcaSlicer/discussions/831#discussioncomment-6274332
- https://manual.slic3r.org/advanced/flow-math

**Reply by RandomGgames** [reply]
Everything you linked only talks about the first layer though. I have experimented a lot with different elephant foot compensation values, most slicers only effect the first layer though which isn't enough for my printer.

**Reply by z28z34man** [reply]
I would like to see this as well.  Seeing as OrcaSlicer and SuperSlicer are both based off of PrusaSlicer it shouldn't be too hard to merge.

**Reply by z28z34man** [reply]
That was quick.  it looks like it is in the new beta.

**Reply by punkweazel** [reply]
I found a bug with this... It now cancels out overhang slow-down, which is something that I also like to have on the first few layers...
https://github.com/SoftFever/OrcaSlicer/issues/9823
